wz_strategic_plugin/include/wz_strategic_plugin/wz_strategic_plugin_config.h

```diff
@@ -29,7 +29,7 @@ struct WzStrategicPluginConfig
   double vehicle_decel_limit_multiplier = 0.75;
 
   //! The length parameter of the participant vehicle used to help calculate the distance before stopping at a red traffic signal
-  double vehicle_length = 5.0;
+  double vehicle_length = 0.0;
```

Comment:
It would be better to remove this parameter in its entirety then have it function as a no-op.